mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/bridge/mapping/annotation/FilterFactoryRef.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.bridge.mapping.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.hibernate.search.engine.search.predicate.factories.FilterFactory;
+
+/**
+ * References a {@link FilterFactory}.
+ * <p>
+ * References can use either a name, a type, or both.
+ */
+@Documented
+@Target({}) // Only used as a component in other annotations
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FilterFactoryRef {
```

Comment:
Please remember to remove this annotation once you addressed my comment on `@FullTextFilter`.